contrib/inventory/ec2.py

```diff
@@ -253,12 +254,18 @@ def read_settings(self):
 
         # Regions
         self.regions = []
-        configRegions = config.get('ec2', 'regions')
+        if config.has_option('ec2', 'regions'):
```

Comment:
Instead of doing this everywhere, on instantiation of the `ConfigParser` (or SafeConfigParser) you can pass a dictionary of defaults, so all the `config.get.....` calls need no alteration, you just need to build a dictionary of all the defaults (for `regions`, `regions_exclude`, and so forth. It would mean we don't have defaults sprinkled all over the code, and probably be net fewer lines.